### PR TITLE
chore: Add a pre-release issue template

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
Part of https://github.com/stackabletech/issues/pull/602

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pre-release.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+name: Pre-Release Product Updates
+about: This template can be used to track the product leading up to the next Stackable release
```
Comment (Techassi) [inline .github/ISSUE_TEMPLATE/pre-release.md:3]: note: Seems like there is something missing in that sentence.

```suggestion
about: This template can be used to track the container image updates leading up to the next Stackable release
```

File: .github/ISSUE_TEMPLATE/pre-release.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+name: Pre-Release Product Updates
```
Comment (Techassi) [inline .github/ISSUE_TEMPLATE/pre-release.md:2]: suggestion:

```suggestion
name: Pre-Release Container Image Updates
```

File: .github/ISSUE_TEMPLATE/pre-release.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+about: This template can be used to track the product leading up to the next Stackable release
+title: Release XX.(X)X
```
Comment (Techassi) [inline .github/ISSUE_TEMPLATE/pre-release.md:4]: suggestion:

```suggestion
title: "chore: Update Container Images for Stackable Release XX.(X)X"
```

File: .github/ISSUE_TEMPLATE/pre-release.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+labels: ['epic']
+assignees: ''
+---
+
+<!--
+    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
+    This was created by an issue template: https://github.com/stackabletech/issues/issues/new/choose.
+-->
+
+## Product Updates for Stackable Release XX.(X)X
```
Comment (Techassi) [inline .github/ISSUE_TEMPLATE/pre-release.md:14]: suggestion:

```suggestion
## Container Image Updates for Stackable Release XX.(X)X
```

**Review by Techassi — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/pre-release.md (hunk 3 above)
Comment (NickLarsenNZ) [reply .github/ISSUE_TEMPLATE/pre-release.md:4]: Oh I like this!

**Review by Techassi — APPROVED** [review]

**Pull request merged**